src/components/TasksTable/index.jsx

```diff
@@ -92,33 +111,59 @@ export default class TasksTable extends Component {
     });
   };
 
+  handleChipClick = tag => () => {
+    const query = this.getQuery();
+    const newQuery =
+      query.tag === tag ? omit(['tag'], query) : { ...query, tag };
+
+    this.props.history.push({
+      search: `?${stringify(newQuery)}`,
+    });
+  };
+
   render() {
     const { items, classes } = this.props;
-    const { sortBy, sortDirection } = this.getQuery();
-    const data = this.getTableData(sortBy, sortDirection, items);
+    const iconSize = 14;
+    const query = this.getQuery();
+    const { sortBy, sortDirection, tag } = query;
+    const data = this.getTableData(sortBy, sortDirection, tag, items);
 
     return (
       <div className={classes.tableWrapper}>
         <DataTable
           title="Bugs & Issues"
           items={data}
           renderRow={item => (
-            <TableRow
-              hover
-              target="_blank"
-              rel="noopener noreferrer"
-              tabIndex={-1}
-              key={item.summary}
-              component="a"
-              href={item.url}
-              className={classes.link}>
+            <TableRow tabIndex={-1} key={item.summary}>
               <TableCell component="th" scope="row">
                 {item.project}
               </TableCell>
-              <TableCell className={classes.summary}>{item.summary}</TableCell>
+              <TableCell>
+                <List className={classes.summary}>
+                  <ListItem
+                    classes={{
+                      gutters: classes.summaryItem,
+                    }}
+                    button
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    component="a"
+                    href={item.url}>
+                    <ListItemText>{item.summary}</ListItemText>
+                    <LinkIcon size={iconSize} />
+                  </ListItem>
+                </List>
+              </TableCell>
               <TableCell className={classes.tags}>
                 {item.tags.map(tag => (
-                  <Chip key={tag} label={tag} className={classes.chip} />
+                  <Chip
+                    key={tag}
+                    label={tag}
+                    className={classNames({
+                      [classes.clickedChip]: tag === query.tag,
+                    })}
+                    onClick={this.handleChipClick(tag)}
```

Comment:
For every tag, this is invoking `handleChipClick(tag)` on page load. Another way to do this is by adding a name attribute `<Chip name={tag} onClick={this.handleTagClick} ...>` and changing `handleChipClick` to be:

```jsx
handleTagClick = ({ currentTarget }) => {
  const tag = currentTarget.getAttribute('name');
  const query = this.getQuery();
  const newQuery =
    query.tag === tag ? omit(['tag'], query) : { ...query, tag };

  this.props.history.push({
    search: `?${stringify(newQuery)}`,
  });
};
```

Doing it like that will only trigger that function when a tag is actually clicked.